Steer/DigitizerWorkflow/src/SimpleDigitizerWorkflow.cxx

```diff
@@ -162,6 +163,9 @@ bool wantCollisionTimePrinter()
 
 std::shared_ptr<o2::parameters::GRPObject> readGRP(std::string inputGRP = "o2sim_grp.root")
 {
+  // init magnetic field
+  o2::Base::Propagator::initFieldFromGRP(inputGRP);
```

Comment:
ok ... so this will actually load the magnetic field for everyone which we need to be aware off. But it might not be such a bad idea if we manage to do only once before forking off.

Reply:
Can it stay for the moment or should it be moved ?

Reply:
yes it can stay. I will follow up putting this into the right place later.